conans/__init__.py

```diff
@@ -4,7 +4,7 @@
 from conans.client.build.cmake import CMake
 from conans.client.toolchain.cmake import CMakeToolchain
 from conans.client.toolchain.make import MakeToolchain
-from conans.client.toolchain.msbuild import MSBuildToolchain
+from conans.client.toolchain.msbuild import MSBuildToolchain, MSBuildCmd
```

Comment:
The name of the tool is important. We could dynamically change the MSBuild helper as we do with CMake, but I am not convinced about that approach, so I am suggesting a new helper. But we can discuss.